### ISSUE TITLE
Multi-user failure of Keybase filesystem on MacOS

### ISSUE THREAD

**Issue by kastork** [keybase/kbfs#533]
Keybase GUI Version: 1.0.18-20161116190722+95f8303
This is MacOS El Capitan with all updates.

It seems that only one user (the first one to try) on a macOS system is allowed to access the Keybase File System.

1. User 1, and "admin" user installs keybase and gets set up (new Keybase user).  Attach the Mac as a device using the name of the Mac as the device name.
2. User 1 follows User 2 on Keybase using the command line.
3. User 1, on the command line, cd's to `/keybase/private/<user1><user2>`
4. Fast user switch (to User 2's account on the Mac), User 2 is existing Keybase user that is all set on other devices.
5. User 2 starts the Keybase application, logs in.  Add the mac as a device using the name of the Mac as the device name.
6. User 2 goes to the command line and follows User 1 on keybase
7. User 2 attempts to cd to  `/keybase/private/<user1><user2>`.
7.a. User 2 receives "Operation not permitted"
8. User 2 attempts to cd to  `/keybase/private/<user2><user1>`.
8.a. User 2 receives "Operation not permitted"
9. User 2 executes `keybase install`
9.a. receive `ERROR stat /keybase: no such file or directory (kbfs) (ERROR/1804)`
10. Fast user switch back to User 1 on macOS, log out
11. Log in as User 2 on mac
12. `keybase login`
13. `keybase install`
```
INFO Starting keybase.service
ERROR stat /keybase: no such file or directory (kbfs) (ERROR/1804)
```

**Comment by strib** [comment]
Correct, that is intentional for now.  We have a plan to support multiple users on the same system, but not sure when we'll get to it.  Please install Keybase as the user you plan to use it as.  Sorry for the inconvenience!

**Comment by kastork** [comment]
Okay, thanks for clarifying.

**Comment by mbauhardt** [comment]
any news here?

**Comment by jzila** [comment]
We're working on a solution to address this.
